docs/Reference/Pantheon-API-Methods.md

```diff
@@ -904,6 +904,75 @@ Returns the account balance of the specified address.
     }    
     ```
     
+### eth_getProof
+
+Returns the account and storage values of the specified account, including the merkle proof.
+
+**Parameters**
+
+`DATA` - 20-byte address of the account or contract.
+
+`ARRAY` - 32-byte storage values of the account or contract address as described in [eth_getStorageAt](#eth_getstorageat)
+
+`QUANTITY|TAG` - Integer representing a block number or one of the string tags `latest`, `earliest`, or `pending`, as described in [Block Parameter](../HowTo/Interact/Pantheon-APIs/Using-JSON-RPC-API.md#block-parameter).
+
+**Returns**
+
+`result`: *Object* - Account details:
+
+* `balance`:`Quantity` - Account balance.
```

Comment:
I'd leave the fullstop off the end of the lines (because I was here first, my fullstop minimal approach wins ;-))